BlogTemplate/Pages/Post.cshtml.cs

```diff
@@ -21,6 +26,12 @@ public PostModel(BlogDataStore dataStore)
 
         public Post Post { get; set; }
 
+        public HtmlString HtmlBody()

```

Comment:
You may need to rebase this branch, or merge from master again.  I think @julietdaniel  refactored this code already.